### DISCUSSION TITLE
LTSP 19+ improvements

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#300] [Q&A]
Hi All,

I am currently documenting the improvements of the new LTSP 19.08, comparing it with the previous version LTSP 5.2, however, I have noticed the differences of memory footprint but I can not seem to get **the exact differences of file sizes in MB.** I tried to run the _apt show_ but the file size says unknown. 
-   iPXE, including multiple booting images
- new technologies like Wayland updated desktop environments etc
- maintaining the virtual client image using a virtualization player

PLEASE do comment if I also left out anything.

**Comment by alkisg** [comment]
Hi tzvidzie,

what do you mean "differences of memory footprint"?

```shell
# apt show ltsp
Installed-Size: 265 kB
```

The new ltsp is just a few KB, a lot smaller than the old one, and it has no running services like the old one (e.g. ldminfod), so both the disk and ram footprints are lower.

Do you mean "how much bigger is the new gnome that supports wayland, compared to the old gnome that only supported xorg"?
That would be a question for gnome, but in general you can see that by comparing the ubuntu 20.04 live cd size to the ubuntu 18.04 live cd size. But the new ltsp runs on older distributions as well, it's not related to the live cd size.

**Comment by tichzvidzayi** [comment]
Hi Alkisg,

Thanks with your response I have noticed the difference but when I run "apt show ltsp-server-standalone" for LTSP5 I get 73.7 KB and for LTSP19 I get the 265 KB. I expected the LTSP5 to be bigger. So I figured I must be getting the size of LTSP5 wrong.

Regards.

**Comment by alkisg** [comment]
ltsp-server-standalone is a metapackage without code. It's only there to get the dependencies, for ease of installation.

You need to add the sizes of all its dependencies, like ltsp-server, ldm-server, ltspfs, and the client packages, ltsp-client-core, ldm, ltspfsd etc.

While the new ltsp package contains all the related code, there are no ldm* or ltspfs* packages anymore.

**Comment by tichzvidzayi** [comment]
Many thanks, that's very helpful
